tests/timeseries/_battery_pool/test_battery_pool_control_methods.py

```diff
@@ -89,6 +89,11 @@ async def _patch_battery_pool_status(
         If `battery_ids` is not None, the mock will always return `battery_ids`.
         Otherwise, it will return the requested batteries.
         """
+        mocker.patch.object(
+            timeseries.battery_pool._methods,  # pylint: disable=protected-access
+            "WAIT_FOR_COMPONENT_DATA_SEC",
+            0.1,
+        )
```

Comment:
Why do we still have some other waiting for data as a simple sleep? Shouldn't that wait be removed too?

Reply:
This is for the metric streaming in the battery pool.  It won't stream any metrics until it has data for all the batteries, etc. So we don't send a new value for capacity or soc as we get data from more and more batteries.

This is independent of the power distributor.